scripts/globals/mobskills/Medusa_Javelin.lua

```diff
@@ -16,16 +16,23 @@ end;
 
 function onMobWeaponSkill(target, mob, skill)
 
+	local mobID = mob:getID();
 	local numhits = 1;
 	local accmod = 1;
 	local dmgmod = 1.5;
+	if (mobID == 16912876) then 

```

Comment:
@teschnei I think we have some pre existing skills in trunk done by mobID that already got merged, I'm going to go find and correct them right now so future editors don't see them and try to do same thing.